Source/Engine/AGS.Engine/Audio/ALAudioClip.cs

```diff
@@ -23,13 +24,16 @@ public ALAudioClip(string id, ISoundData soundData, IAudioSystem system, IAudioE
 			_errors = errors;
             _backend = backend;
 			ID = id;
-			_buffer = new Lazy<int> (() => generateBuffer());
+            _duration = AudioHelpers.GetDuration(soundData.DataLength, soundData.Channels, soundData.BitsPerSample, soundData.SampleRate);
```

Comment:
Why do we need AudioHelpers? Why not just have it as a method in ALAudioClip? Do you think this is something people will want to call outside of ALAudioClip?